tests/test_pipeline_conversion.py

```diff
@@ -50,3 +50,18 @@ def test_pipeline_conversion_override_inputs(pipeline_overridden_config: Config)
     assert overridden["template"]["inputs"].get('training-images', [])[0] == 'overridden node image'
     assert len(overridden["template"]["inputs"].get('training-images', [])) == 1
     assert len(overridden["template"]["parameters"].items()) == 3
+
+
+def test_pipeline_parameter_conversion(pipeline_with_parameters_config):
+    for _name, pipe in pipeline_with_parameters_config.pipelines.items():
+        result = PipelineConverter(
+            config=pipeline_with_parameters_config,
+            commit_identifier="latest",
+        ).convert_pipeline(pipe)
+        assert isinstance(result["parameters"], dict)
+        parameter_name = "id"
```

Comment:
This assign here inside the loop makes me wonder why we need to set it every iteration so what is changing it and I can't find any. As it seems to be constant it could be initialized once out side of the loop.

Reply:
makes sense